### PR TITLE
Update BizDev role

### PR DESCRIPTION
**Pull request by gabrieldamours**

### CODE REVIEW

File: html/opencraft_team_roles_2.html [hunk 1]
```diff
@@ -152,18 +152,37 @@ <h3>Admin</h3>
...
 <ul>
+  <li>
+    Handling contact@ email
+  </li>
   <li>
     Looking out for business opportunities
   </li>
   <li>
-    Gathering needs and drafting quotes to leads
+  Meeting with prospects and gathering their requirements
```
Comment (antoviaque) [inline html/opencraft_team_roles_2.html]: Indentation issue :)

File: html/opencraft_team_roles_2.html [hunk 2]
```diff
@@ -152,18 +152,37 @@ <h3>Admin</h3>
...
   <li>
     Handling coordination between client and the rest of the team
   </li>
+  <li><a href="https://docs.google.com/drawings/d/12GXlLwWgb3qBB-iDjHJJegV1qqHJ6Fru6RVjhgtj6IQ/edit?usp=sharing">OpenCraft lead management flowchart</a>
+  </li>
+
+  <li> 
+  <a href="https://docs.google.com/drawings/d/15RceotA5oqKNr0gQCR924B9OnbY5xy6E7b2xezaIkyc/edit?usp=sharing">OpenCraft client onboarding flowchart</a>
+  </li>
+
```
Comment (antoviaque) [inline html/opencraft_team_roles_2.html]: @gabrieldamours Maybe also include the responsibilities that comes out of BIZ-91, ie the handling/support of self-hosted clients?

**Comment by gabrieldamours** [comment]
Hey @antoviaque , I made and committed the changes you requested but I'm not sure I'm doing this the right way.

File: html/opencraft_team_roles_2.html [hunk 3]
```diff
@@ -159,7 +159,7 @@ <h3>Business developer</h3>
     Looking out for business opportunities
   </li>
   <li>
-  Meeting with prospects and gathering their requirements
+   Meeting with prospects and gathering their requirements
```
Comment (antoviaque) [inline html/opencraft_team_roles_2.html]: Still missing one more space :p

**Pull request merged**